src/main/java/frc/robot/subsystems/drive/Drivetrain.java

```diff
@@ -216,7 +232,7 @@ public void drive(Vector spdVec, Vector rotVec, boolean altDrive, boolean center
    */
   public void driveChassisSpeeds(ChassisSpeeds spds) {
     Vector spd = new Vector(spds.vxMetersPerSecond, spds.vyMetersPerSecond);
-    spdCommanded = spd;
+    spd.div(DriveConfig.kMaxSpeedMetersPerSecond);
```

Comment:
This probably isn't necessary since you desaturate the wheel speeds anyway. I recommend testing this though.